Part_One/PartOne_WeatherData.py

```diff
@@ -0,0 +1,71 @@
+
+import json
+
+WEATHER_FILE_NAME = 'weather.json'
+NOT_VAlID_NAME_MSG = 'Not valid name or integers for'
+DAY_OUTPUT_MSG = 'The day number with the smallest temperature spread is'
+NOT_VALID_ROW_MSG = 'There is not a valid row in the file'
+EMPTY_STRING = ''
+
+
+# The weather object
+class WeatherObject(object):
+    def __init__(self, day_number, maximum_temp, minimum_temp):
+        self.m_day_number = day_number
+        self.m_spread_temp = abs(maximum_temp - minimum_temp)
+
+
+# JSON object decoding - decode a JSON object into a WeatherObject type.
+def object_weather_decoder(obj):
+    day = obj['Dy']
+    max_t = obj['MxT']
+    min_t = obj['MnT']
+    if check_validity_of_the_object(obj, day, max_t, min_t):
+        return WeatherObject(day, int(max_t), int(min_t))
+    else:
+        print("{} {}".format(NOT_VAlID_NAME_MSG, obj))
+        return None
+
+
+# Check if the 3 fields of the jason file are valid
+def check_validity_of_the_object(obj, day, max_t, min_t):

```

Comment:
Function and variable names should be descriptive, but not too long.
try to reduce long function names to shorter but still descriptive names.
And try to use comments if you fill it's not clear enough.